.travis.yml

```
@@ -38,9 +38,9 @@ jobs:
     - <<: *build-platform
       env: PLATFORM='tgl'
 
-    - name: "./scripts/build-tools.sh"
+    - name: "./scripts/build-tools.sh Release"
       before_install: *docker-pull-sof
-      script: ./scripts/docker-run.sh ./scripts/build-tools.sh
+      script: CMAKE_BUILD_TYPE=Release ./scripts/docker-run.sh ./scripts/build-tools.sh
```

why not use something like this?
```suggestion
      script: ./scripts/docker-run.sh “CMAKE_BUILD_TYPE=Release ./scripts/build-tools.sh"
```

---

I tried this but the docker command does not start a shell. So it would have to be much more complicated with nested quoting issues, something like `docker-run.sh 'bash -c "CMAKE_BUILD_TYPE=Release ./scripts/build-tools.sh'`